vendor/github.com/elastic/ecs/code/go/ecs/version.go

```diff
@@ -20,4 +20,4 @@
 package ecs
 
 // Version is the Elastic Common Schema version from which this was generated.
-const Version = "1.0.1"
+const Version = "1.1.0"
```

Comment:
@andrewkroh Will the events have `ecs.version=1.1.0` based on this? Or is there another constant to changed somewhere?

Reply:
I had done a grep for 1.0.1 in libbeat and didn't find anything so it must be using a constant. We can double check this after we have a build.